packages/content/src/Application/PropertyResolver/PropertyResolverProvider.php

```diff
@@ -15,7 +15,13 @@
 
 use Sulu\Content\Application\PropertyResolver\Resolver\PropertyResolverInterface;
 
-class PropertyResolverProvider
+/**
+ * @internal The constructor of this class maybe change in future releases to add new features or improve performance.
+ *           Use this service via the dependency injection container via the PropertyResolverProviderInterface only.
+ *
+ * If you need to override this service use service decoration via the provided interface: https://symfony.com/doc/6.4/service_container/service_decoration.html.
```

Comment:
```suggestion
 * If you need to override this service use service decoration via the provided interface: https://symfony.com/doc/current/service_container/service_decoration.html.
```

I think we can use `current` as a version in the url

Reply:
We should not use `current` as that can end up in Dead links so I prefer current LTS which is mostly longest keeped uptodate.